client/state/ui/selectors/get-selected-site-id.js

```diff
@@ -3,7 +3,6 @@ import 'calypso/state/ui/init';
 /**
  * Returns the currently selected site ID.
  *
- * @param  {Object}  state Global state tree
```

Comment:
I find it odd that we're removing these; we are actually returning an object here. Perhaps we should change it to return `AnyAction`?